.github/workflows/lint-pipelines.yml

```diff
@@ -0,0 +1,20 @@
+name: Lint Deployment Pipelines
+
+on:
+    pull_request:
+    push:
+        branches: [main, test-me-*]
```

Comment:
- We do not have a `main` branch, this should be `master`. 
- Also, we require these deployments from all PR branches in Relay, so should we be running it there, too?
- We usually use a `/` as pattern, so this would be `test/pipeline-*` for instance.

Reply:
Corrected. re #2: pull_request: will run on everything by default, no?

Reply:
Right, I had missed `pull_request:` above. That's what we're looking for. It runs on every commit that is pushed to an open pull request, as well as the last commit of a branch when a pull request is opened on it.